README.md

```diff
@@ -4,36 +4,28 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-# Vision
-A computer vision library dedicated for auto-driving, robotics and on device applications.
 
 # Mission
 
-KerasCV is a layered repository consisting of core components and modeling components.
+KerasCV is a computer vision library of modular computer vision oriented Keras components.
+These components consist of models, layers, metrics, losses, callbacks, and utility functions.
 
-On the core components, it is made of modular building blocks (ops, functions, layers, metrics, losses, callbacks) that standardizes APIs for computer vision concepts such as data-augmentation pipeline, bounding boxes, keypoints, point clouds, feature pyramid network, etc, so applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art
-training and inference pipelines for common tasks such as image classification, object detection and segmentation, image data augmentation, etc.
+The goal of the library is to provide standardized Keras native APIs for common computer vision tasks such as data-augmentation, classification, object detection, image generation, and more.
```

Comment:
below we state
"""
KerasCV's primary goal is to provide a coherent, elegant, and pleasant API to train state of the art computer vision models.
""

which one is it? Maybe we should merge the two